<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_8">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_7">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <widget class="QLabel" name="labelid">
         <property name="text">
          <string>Employee id</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditid"/>
       </widget>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_2company">
         <property name="text">
          <string>Company</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2cname"/>
       </widget>
       <widget class="QSplitter" name="splitter_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_3Ename">
         <property name="text">
          <string>Employee Name</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3ename"/>
       </widget>
       <widget class="QSplitter" name="splitter_4">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_4Department">
         <property name="text">
          <string>Department</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxDept"/>
       </widget>
       <widget class="QSplitter" name="splitter_5">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_5joiningdate">
         <property name="text">
          <string>Joining date</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_5Jdate"/>
       </widget>
       <widget class="QSplitter" name="splitter_6">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_6Designation">
         <property name="text">
          <string>Designanation</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_6designation"/>
       </widget>
       <widget class="QPushButton" name="AddBtn">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="editTriggers">
        <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
